requirements.txt

```diff
@@ -1,5 +1,5 @@
 requests>=2.12.4,<3            # api tests
 selenium>=2.53.6,<4            # web tests
-Appium-Python-Client>=0.24,<1  # mobile tests
+Appium-Python-Client>=1.3.0,<2 # mobile tests
```

Comment:
if you have checked that toolium is compatible with appium 1.3.0, then the new appium dependencies should be 
`Appium-Python-Client>=0.24,<2`